CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 0.10.2 (2019-10-31)
```

Comment:
Thanks for adding a changelog entry – however, let's add the release number and the date here only once we do the release.

```suggestion
# Unreleased
```